temporalcli/commandsmd/commands.md

```diff
@@ -560,533 +1300,836 @@ Includes options set for [overlap-policy](#options-set-for-overlap-policy).
 Includes options set for [shared-workflow-start](#options-set-for-shared-workflow-start).
 Includes options set for [payload-input](#options-set-for-payload-input).
 
-### temporal server: Run Temporal Server.
+### temporal server: Run Temporal Server
 
-Start a development version of [Temporal Server](/concepts/what-is-the-temporal-server):
+Run a development [Temporal Server](/clusters) on your local system.
+View the Web UI for the default configuration at http://localhost:8233:
 
-`temporal server start-dev`
+```
+temporal server start-dev
+```
+
+Add persistence for Workflow Executions across runs:
 
-### temporal server start-dev: Start Temporal development server.
+```
+temporal server start-dev \
+    --db-filename path-to-your-local-persistent-store
+```
 
-Start [Temporal Server](/concepts/what-is-the-temporal-server) on `localhost:7233` with:
+Set the port from the front-end gRPC Service (7233 default):
+
+```
+temporal server start-dev \
+    --port 7000
+```
```

Comment:
I suspect if people want to change one port, they'll want to change all three, so it might be good to show a single command setting all three:

```
temporal server start-dev \
    --port 7234 \
    --ui-port 8234 \
    --metrics-port 57271
```

Reply:
Great suggestion, seriously!

Reply:
Question, is --ui-port with port + 1000 redundant?